src/resources/Subscriptions.ts

```diff
@@ -132,26 +131,41 @@ export interface SubscriptionUpdateParams<T extends Metadata = Metadata> {
      */
     firstChargeCaptureAfter?: string;
     firstChargeAuthorizationOnly?: boolean;
-}
+};
 
 export type PaymentUpdateParams = Partial<ScheduledPaymentItem>;
 
-export interface ScheduleSettings {
+export type ScheduleSettings = {
     startOn?: string;
     zoneId: string;
     preserveEndOfMonth?: boolean;
-}
+};
 
 /* Response */
-export interface SubscriptionItem<T extends Metadata = Metadata> {
+
+type PeriodParams =
+    | {
+          /**
+           * Can not be used with the cyclicalPeriod parameter
+           */
+          period: SubscriptionPeriod;
```

Comment:
should we add something like 

```
 cyclicalPeriod?: never;
 ```
 here too? Or not so needed since we'll switch to using `cyclicalPeriod ` more from now on?

Reply:
This is not needed as the error will be triggered as it will fall in the other type when the `cyclicalPeriod` is present. Just decided that the error should be on the period rather than the cyclical period.

Reply:
alright yeah that sounds good then 👍 Resolving!